### DISCUSSION TITLE
It's there a way to disable the error sounds?

### DISCUSSION THREAD

**Discussion by nothingbutlucas** [Slackadays/Clipboard#164] [Q&A] [answered]
Hi. I didn't find a way to actually silence the errors.
Is it somewhere in the wiki or readme and I missed it?
Thanks

**Comment by Slackadays** [comment, marked as answer]
Set `CLIPBOARD_NOAUDIO` to 1 to disable the audio (search for "noaudio" in https://github.com/Slackadays/Clipboard for an example)

**Reply by nothingbutlucas** [reply]
Oh audio, I was searching for sound, alert... sorry and thanks!
